extensions/smallrye-openapi-common/deployment/src/main/java/io/quarkus/smallrye/openapi/common/deployment/SmallRyeOpenApiConfig.java

```diff
@@ -70,12 +70,24 @@
     @ConfigItem(defaultValue = "true")
     public boolean autoAddTags;
 
+    /**
+     * This will automatically add security based on the security extension included (if any).
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean autoAddSecurity;
+
     /**
      * Add a scheme value to the Basic HTTP Security Scheme
      */
     @ConfigItem(defaultValue = "basic")
     public String basicSecuritySchemeValue;
 
+    /**
+     * Add a scheme value to the Form HTTP Security Scheme
+     */
+    @ConfigItem(defaultValue = "form")
```

Comment:
I don't see where it is referred to in the PR

Reply:
A, sorry, yes this needs to be removed, this was part of my Form support attempt